turn.rb

```diff
@@ -0,0 +1,55 @@
+class Turn
+	attr_accessor :player, :score, :no_of_dices, :end_of_turn
+
+	def initialize(player)
+		@player = player
+		@score = 0
+		@no_of_dices = 5
+	end
+
+	def start()
+		roll()
+	end
+
+	private
+
+	def roll()
+		roll = Roll.new(@no_of_dices)
+		puts roll.to_s + "\n"
+		
+		if (roll.score() == 0)
```

Comment:
Can use Ruby methods for readability
```suggestion
		if (roll.score.zero?)
```

Reply:
done, committed